pkg/profile/artifact_test.go

```diff
@@ -157,6 +65,81 @@ var _ = Describe("Profile", func() {
 		var err error
 		rootDir, err = ioutil.TempDir("", "test_make_artifacts")
 		Expect(err).NotTo(HaveOccurred())
+		gitRepositoryName = "git-repo-name"
+		gitRepositoryNamespace = "git-repo-namespace"
+		artifacts := []artifact.Artifact{
```

Comment:
do we need two resources here? could we get away with just 1 for slimming the test down? Or is it testing some functionality differently

Reply:
I guess we don't NEED two. :) But it's nice to have more than one if you have a loop. To make sure that the system behaves consistently on each cycle. That there aren't any things that it reuses incorrectly, or caches, or keeps something that its not supposed to.